src/main/java/blackjack/domain/Cards.java

```diff
@@ -0,0 +1,64 @@
+package blackjack.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cards {
+
+    private static final int ADDITION_ACE_SCORE = 10;
+    private static final int LIMITED_ACE_SCORE = 11;
+    private static final int REFERENCE_POINT = 21;
+    private static final String CARD_SIZE_ZERO_ERROR = "카드가 존재하지 않습니다.";
+    private static final int FRONT = 0;
+
+    private final List<Card> cards;
+
+    public Cards(final List<Card> cards) {
+        this.cards = cards;
+    }
+
+    public int size() {
+        return cards.size();
+    }
+
+    public int totalScore() {
+        final int score = cards.stream()
+            .mapToInt(Card::getCardScore)
+            .sum();
+
+        if (hasAce() && score <= LIMITED_ACE_SCORE) {
+            return score + ADDITION_ACE_SCORE;
+        }
+        return score;
+    }
+
+    public void add(final Card drawCard) {
+        cards.add(drawCard);
+    }
+
+    private boolean hasAce() {
+        return cards.stream().anyMatch(Card::isAce);
+    }
+
+    public List<String> openCardOne() {
+        if (cards.isEmpty()) {
+            throw new RuntimeException(CARD_SIZE_ZERO_ERROR);
+        }
+        return Arrays.asList(cards.get(FRONT).getCardName());
```

Comment:
```suggestion
        return List.of(cards.get(FRONT).getCardName());
```

https://bepoz-study-diary.tistory.com/349

Reply:
프로젝트에서 사용하는 자바 버전이 `List.of`를 사용할 수 없었는데 `Arrays.asList`를 사용했을 때와의 차이가 있네요 ㅠㅠ